app/main/forms.py

```diff
@@ -64,15 +64,18 @@
 from app.models.feedback import PROBLEM_TICKET_TYPE, QUESTION_TICKET_TYPE
 from app.models.organization import Organization
 from app.utils import branding, merge_jsonlike
+from app.utils.csv import get_user_preferred_timezone
 from app.utils.user_permissions import all_ui_permissions, permission_options
 
 
 def get_time_value_and_label(future_time):
+    et = pytz.timezone(get_user_preferred_timezone())
```

Comment:
D'oh same for these, sorry for not flagging them before too!

```suggestion
    preferred_tz = pytz.timezone(get_user_preferred_timezone())
```